PAPER/FIT_Paper_Ex1/Part_5.tex

```diff
@@ -1,79 +1,86 @@
-and the precision level to reach.
+The FIT project approach consists in testing code functionality by an intra-code
+comparison of calculation with and without the feature. The FIT project is not a
+benchmark and there is no inter-simulator comparison but aims to perform
+intra-simulator comparison, comparing the simulator output obtain with a single
+simulator, with and without the feature/functionality to test. The strength of
+this project relies on the diversity of simulator used to solve the different
+problems allowing to reach nearly simulator-agnostic conclusion. Following this
+principle, the FIT project will provide informations about which functionalities
+are required according to the question that needs to be answered and the
+precision level to reach.
 
-The work presented here is dedicated fuel cycle simulators ability to adapt
-fresh fuel compositions to the available material. Two treatments are
-considered: a fixed enrichment for fresh fuel, whatever the isotopic composition
-or an enrichment calculation to adapt the fresh fuel composition in accordance
-to the isotopic composition. The first exercise treated here focuses on
-potentiality studies to recycle plutonium and the estimators built are
-consistent with this kind of studies.  
+The work presented here is dedicated fuel cycle simulators ability to vary fresh
+fuel compositions according to the available materials. Two treatments have been
+considered: a fixed enrichment for fresh fuel, regardless of the isotopic
+composition, and a model to adjust the fresh fuel composition in accordance
+to the isotopic composition. In this first exercise, the output estimators focus
+on availability of the plutonium in the fuel cycle, so will probably be more
+valuable for fuel cycle study involving plutonium recycling.
 
-In agreement with the FIT philosophy all calculations were performed with a
+In agreement with the FIT philosophy, all calculations were performed with a
 fixed fraction and with the use of a model that calculate fissile enrichment for
-fresh fuels. No code to code comparison are done here but each code is tested
-with and without a fixed fraction. The importance of adapting fresh fuel
-compositions is tested here for 3 main questions : the global plutonium
-inventory estimation, the global plutonium inventory evolution rate and the
-local plutonium inventory in spent fuel to be reprocessed. 4 different codes are
-tested for PWR (CLASS, ANICCA, TR\_EVOL and CYCLUS) and 3 for SFR (CLASS,
-TR\_EVOL and JOSETTE). 
+fresh fuels. Inter-software comparison have been done but each code is tested
+against itself, \gls{FF} vs \gls{FLM} on 2 different reactor, across 5 different
+simulators, (CLASS, ANICCA, TR\_EVOL and CYCLUS) for the \gls{PWR} and (CLASS,
+TR\_EVOL, JOSETTE) for the \gls{SFR}. The importance of adjusting fresh fuel
+compositions have been tested around 3 main questions : 
+\begin{itemize}
+    \item the global plutonium inventory estimation,

```

Comment:
Not sure to agree with those three item. For me, item 1 and 2 are quite similar. And for the item 3, why focus on spent fuel...? Since all the spent fuel front end will be impacted.